package.json

```diff
@@ -62,12 +62,12 @@
     "less-loader": "^5.0.0",
     "style-loader": "^1.2.0",
     "stylelint": "^12.0.1",
-    "stylelint-config-wikimedia": "^0.8.0",
+    "stylelint-config-wikimedia": "^0.10.1",
```

Comment:
This is a good idea. If you submit a PR just for that I'll accept it.

Reply:
I would be grateful.I have already submitted the Pull request for this (this string is all about that only) I would be more than happy if you merge it.

Reply:
This PR has 3 changes. If you create another one with only 1 change: the stylelint config upgrade, I will approve it.

The other points require more discussions so bundling them in the same PR is not ideal.

Reply:
Okay I am making an PR with only the stylelint-config upgrade other than than that every thing remains same.. right?

Reply:
I have made the changes as per your request please see to it!

Reply:
Could you update the PR description? It is still about the instrumentation variable.

Reply:
Yes of course. Doing in a minute!

Reply:
Done!